tests/benchdnn/graph/utils.hpp

```diff
@@ -146,7 +146,7 @@ dnnl::graph::op::attr attrstr2kind(const std::string &attr_name);
 const std::string &attrstr2type(const std::string &attr_name);
 
 std::string get_default_tag(size_t length);
-std::string strides2memory_tag(const size_t ndims,
+std::string strides2memory_tag(const dnnl::graph::logical_tensor::dims &dims,
```

Comment:
Do you want to mention this function doesn't consider non-contiguous strides. e.g. shape=[1,2,3,4], stride=[60,30,10,1]